tasks/ast_tools/src/derives/estree.rs

```diff
@@ -25,12 +25,17 @@ impl Derive for DeriveESTree {
     fn derive(&mut self, def: &TypeDef, _: &LateCtx) -> TokenStream {
         let ts_type_def = match def {
             TypeDef::Enum(def) => typescript_enum(def),
-            TypeDef::Struct(def) => typescript_struct(def),
+            TypeDef::Struct(def) => Some(typescript_struct(def)),
         };
-        let ts_type_def = quote! {
-            #[wasm_bindgen::prelude::wasm_bindgen(typescript_custom_section)]
-            const TS_APPEND_CONTENT: &'static str = #ts_type_def;
+        let ts_type_def = if let Some(ts_type_def) = ts_type_def {
```

Comment:
Maybe Option::map_or would make this easier to read? I think the implementation is fine especially since it'll be temporary.

Reply:
My personal preference is more explicit code, rather than the sometimes obscure `Option` methods. But that's just a matter of personal taste. Going to leave it as it is for now - I don't think a big deal either way.